projects/demos/app/demo-modules/day-view-scheduler/day-view-scheduler.component.ts

```diff
@@ -83,7 +83,8 @@ export class DayViewSchedulerCalendarUtils extends CalendarUtils {
   templateUrl: 'day-view-scheduler.component.html',
   providers: [DayViewSchedulerCalendarUtils],
 })
-export class DayViewSchedulerComponent extends CalendarWeekViewComponent
+export class DayViewSchedulerComponent
```

Comment:
Can we pls avoid style change here ?

Reply:
Hey sure :-). I have push the fix for the code styling.

Reply:
Style have the old styles.